recipes/aenum/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "aenum" %}
+{% set version = "2.0.8" %}
+{% set sha256 = "3209fa41b8c41345442e8d9b5158a57d3e96d84c3d5ebbe8e521e1e2eff1598d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

Comment:
Can this be noarch?

Reply:
It does not look like it does. The setup.py code has a py2 vs py3 check.